packages/integrations/cloudflare/test/fixtures/vite-plugin/wrangler.jsonc

```diff
@@ -1,12 +1,24 @@
 {
 	"compatibility_date": "2025-05-21",
-	"main": "@astrojs/cloudflare/entrypoints/server",
+  "compatibility_flags": [
+  	"nodejs_compat",
```

Comment:
This is necessary for now because image processing depends on Node, even though this code doesn't need to run at runtime (I think). 

I have an issue to see if we can fix this so the compat flag is not needed.